api/rest.go

```diff
@@ -0,0 +1,40 @@
+package api
+
+import (
+	"encoding/json"
+	"github.com/TheChaseBL/pokerepo-backend/database"
+	"github.com/TheChaseBL/pokerepo-backend/entity"
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+)
+
+func AddPokemonHandler(w http.ResponseWriter, r *http.Request) {
+	var p entity.Pokemon
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&p)
+	if err != nil {
+		panic(err)
+	}
+	w.WriteHeader(http.StatusOK)
+	database.Conn.AddEntry(p)
```

Comment:
I suggest we have `AddEntry` return err if the DB operation fails, then we return not `http.StatusOK`.

Reply:
Yeah that's a good idea.